safe/impact_function/test/test_postprocessors.py

```diff
@@ -218,6 +227,57 @@ def test_size_post_processor(self):
         impact_fields = impact_layer.dataProvider().fieldNameMap().keys()
         self.assertIn(feature_value_field['field_name'], impact_fields)
 
+    def test_productivity_post_processors(self):
```

Comment:
__[flake8]__
* [E402] module level import not at top of file

<!-- Comment by [SideCI](https://sideci.com) -->